modules/microshift-nodeport-unreachable-workaround.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-known-issues.adoc
+:_content-type: PROCEDURE
+[id="microshift-nodeport-unreachable-workaround_{context}"]
+= Manually restarting the `ovnkube-master` pod to resume node port traffic
+
+There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped. To troubleshoot this issue, manually restart the `ovnkube-master` pod in the `openshift-ovn-kubernetes` namespace.
```

Comment:
ISG recommends against starting a clause with "There is" so maybe:

s/There is an issue with new installs of {product-title} in which the `NodePort` service traffic might be stopped./New installs of {product-title} might experience stopped `NodePort` service traffic.

Reply:
Saying the software might experience something is anthropomorphism (a different style rule). Will consider a change.